src/WebJobs.Script.Abstractions/Description/Binding/BindingMetadata.cs

```diff
@@ -81,6 +87,7 @@ public static BindingMetadata Create(JObject raw)
             string bindingDirectionValue = (string)raw["direction"];
             string connection = (string)raw["connection"];
             string bindingType = (string)raw["type"];
+            var properties = raw["properties"].ToObject<IDictionary<string, object>>();
```

Comment:
Wondering about casing on the lookups we're doing in this dictionary. Can we make this case insensitive? Though I see the FunctionMetadata Properties dictionary is also declared case sensitive which was an oversight.